src/sentry/shared_integrations/client/base.py

```diff
@@ -72,49 +78,41 @@ def build_url(self, path: str) -> str:
             return f"{base_url}/{path}"
         return path
 
+    def _get_redis_key(self):
+        if not hasattr(self, "integration_id"):
```

Comment:
idk that this needs to be `hasattr` - if it's not passed it'll be `None` so you can just do `if not self.integration_id: ... `

Reply:
Yeah this makes sense to me and I had this earlier but it was causing the Bitbucket test cases to fail so maybe I should do the` if not` check after